src/store/RelayStoreData.js

```diff
@@ -86,10 +84,7 @@ class RelayStoreData {
    * Get the data set backing actual Relay operations. Used in GraphQLStore.
    */
   static getDefaultInstance(): RelayStoreData {
-    if (!_instance) {
-      _instance = new RelayStoreData();
-    }
-    return _instance;
+    return RelayStore.getStoreData();
```

Comment:
there's a circular dependency here:

- `RelayStore` requires `RelayContext` to create the default instance
- `RelayContext` requires `RelayStoreData` to create the underlying data object
- `RelayStoreData` requires `RelayStore` in order to return the default instance

This can break depending on the build system used. What's the motivation for changing the implementation here? Seems like the previous approach would still work.